extensions/amp-video/0.1/amp-video.js

```diff
@@ -534,17 +557,21 @@ class AmpVideo extends AMP.BaseElement {
   /**
    * @param {string} src
    * @param {?string} type
+   * @param {?string} bitrate
```

Comment:
if this parameter is optional: `string=`
But from your map it's a `number`?

Reply:
`string|number=`?